src/GraphQL/Types/Scalars/EnumerationGraphType.cs

```diff
@@ -17,11 +17,17 @@ namespace GraphQL.Types
     /// </summary>
     public class EnumerationGraphType : ScalarGraphType
     {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="EnumerationGraphType"/> class.
+        /// </summary>
         public EnumerationGraphType()
         {
             Values = new EnumValues();
         }
 
+        /// <summary>
+        /// Adds a value to the allowed set of enumeration values.
```

Comment:
The meaning of the arguments needs to be explained here. Especially the difference between `name` (public API surface) and `value` (internal server value).

Reply:
I'm adding more details to EnumValueDefinition.Name and EnumValueDefinition.Value.  Please review.

Reply:
Oh I see this overload could use some more definitions also... I'll make corresponding changes.....

Reply:
After updating those definitions, I copied the descriptions into the param xml comments; take a look.

Reply:
ok